index.vue

```diff
@@ -53,7 +48,8 @@
   import ScbdFooter from './src/components/cbd-nav/footer.vue'
   import CopCountDown from './src/components/countdown/cop-count-down.vue';
   import dateSelector from  './src/components/inputs/dateSelector.vue'
-  import { ref, provide } from 'vue';
+  import kmNotes from  './src/components/inputs/notes.vue'
```

Comment:
the notes must be located in components/controls/ or components/controls/km. not in 'input' it's not a basic field type

Reply:
change to componentss/controls path